lib/oli_web/components/delivery/user_account.ex

```diff
@@ -39,6 +39,29 @@ defmodule OliWeb.Components.Delivery.UserAccount do
   attr(:class, :string, default: "")
   attr(:dropdown_class, :string, default: "")
 
+  def workspace_menu(%{ctx: %{author: %{system_role_id: system_role_id}}} = assigns) when system_role_id in [2, 3, 4] do
```

Comment:
The `is_admin` assign should be set if *any* admin roles are present, so it is a bit strange that it is not picking that up here before this change